.github/CODEOWNERS

```
@@ -15,7 +15,9 @@ Makefile @ComposableFi/sre
 
 ## Parachain related files
 
-runtime/ @ComposableFi/parachain-leads
+runtime/ @ComposableFi/parachain-leads @ComposableFi/core
```

No need to add `core` to the runtime ownership, as there is already a big overlap. Core is intended for files where we haven't gotten around to assigning owners.

```suggestion
runtime/ @ComposableFi/parachain-leads
```